docs/findings/el7.md

```diff
@@ -7,199 +7,89 @@
-
-### `/tmp` — `noexec` Option Missing
-
-This is a false-finding. Many scanning tool assume that `/tmp` will be a standard, disk-based filesystem. The OSes typically hardened via Watchmaker do not use a standard, disk-based filesystem for `/tmp`. Instead, `/tmp` is implemented as a pseudo-filesystem via the `tmp.mount` systemd service.
-
-Because scanners typically assume that `/tmp` will be a standard, disk-based filesystem, they will typically scan the contents of the `/etc/fstab` file to ensure compliance. When `/tmp` is managed via the `tmp.mount` systemd service, there will be no entry for the `/tmp` filesystem in the `/etc/fstab` file. This creates a false scan-error. The correct scan-method is to look for the `noexec` option in the `/etc/systemd/system/tmp.mount.d/options.conf` configuration file, instead.
-
-To properly verify that Watchmaker has applied the required mount-option to the systemd-managed `/tmp` mount:
-
-```bash
-grep -E '\s\/tmp\s.*noexec' /proc/mounts
-grep noexec /etc/systemd/system/tmp.mount.d/options.conf
-```
-
-The above verifies that the mounted `/tmp` has the desired mount-option set and that the setting will persist after a reboot.
-
-### `/tmp` — `nosuid` Option Missing
-
-This is a false-finding. Many scanning tool assume that `/tmp` will be a standard, disk-based filesystem. The OSes typically hardened via Watchmaker do not use a standard, disk-based filesystem for `/tmp`. Instead, `/tmp` is implemented as a pseudo-filesystem via the `tmp.mount` systemd service.
-
-Because scanners typically assume that `/tmp` will be a standard, disk-based filesystem, they will typically scan the contents of the `/etc/fstab` file to ensure compliance. When `/tmp` is managed via the `tmp.mount` systemd service, there will be no entry for the `/tmp` filesystem in the `/etc/fstab` file. This creates a false scan-error. The correct scan-method is to look for the `nosuid` option in the `/etc/systemd/system/tmp.mount.d/options.conf` configuration file, instead.
-
-To properly verify that Watchmaker has applied the required mount-option to the systemd-managed `/tmp` mount:
-
-```bash
-grep -E '\s\/tmp\s.*nosuid' /proc/mounts
-grep nosuid /etc/systemd/system/tmp.mount.d/options.conf
-```
+Conditionally-Valid Finding: Smart Card Login use and configuration is site-specific. Site has not provided specification for implementing this setting within scanned context.
 
-The above verifies that the mounted `/tmp` has the desired mount-option set and that the setting will persist after a reboot.
+## Configure the Firewalld Ports
 
-### `/var/tmp` — `nodev` Option Missing
+Invalid Finding: Watchmaker implements setting. Howeve, scanner regex may not be sufficiently-flexible in its specification.
 
-Cannot satisfy: `/var/tmp` not implemented as stand-alone filesystem.
+## Set Default firewalld Zone for Incoming Packets
```

Comment:
I think we can remove this. We'd consider this more a bug/feature to fix, rather than something users should expect as a finding.

Reply:
Once I fix the `ash-linux-formula` content, I'd probably say that this is a conditional: it's a near lock that there's going to be sites that don't want the default zone to be "drop" &mdash; especially if they rely on ping-sweeps for anything (security-related or otherwise).

Reply:
This looks resolved now I think?